frontend/src/views/Orders/Orders.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Route, Switch } from "react-router-dom";
+import OrderTable from '../../components/OrderTable/OrderTable'
+import ViewWrappers from "components/ViewWrappers/ViewWrappers";
+import Order from './Order';
+
+import "./Orders.scss";
+import Yippey from 'components/Yippey/Yippey';
+
+const orderTable = {name: 'Tina Mayer', street: 'Hauptstraße 45', zipcity: '78464 Konstanz', products: [
+  {quantity: 1, name: 'Very great bread indeed', price: 3.45},
+  {quantity: 3, name: 'Very great tea indeed', price: 10.35}
+]};
+
+const Orders = (props) => {
+  const match = props.match;
+
+  return (
+    <div className="Orders">
+      <Switch>
+        {/* an order overview */}
+        <Route path={`${match.path}/overview`}>
+          <ViewWrappers.View container withPadding>
```

Comment:
You can use the View around the entire Switch and drop the `div className="Orders"` then. You can also pass View a className string